keyvi/src/cpp/dictionary/dictionary_compiler.h

```diff
@@ -103,7 +104,12 @@ class DictionaryCompiler
       sorter_.set_available_memory(memory_limit);
       sorter_.begin();
 
-      value_store_= new ValueStoreT(params);
+      if (params_.count(TEMPORARY_PATH_KEY) == 0) {
+        params_[TEMPORARY_PATH_KEY] =
+            boost::filesystem::temp_directory_path().string();
+      }
+
+      value_store_= new ValueStoreT(params_);
```

Comment:
I started trying it out - and agree doing it - it turned out to be a bigger change. Therefore better in a separate PR.